src/UI/Button.tsx

```diff
@@ -0,0 +1,42 @@
+import { styled } from "@mui/material/styles";
+import { Button } from "@mui/material";
+
+const FormButtons = styled(Button)(({ theme, size }) => ({
```

Comment:
Do you need `size` here? I don't see you using it anywhere.

Reply:
I overlooked that.I delete size..it's done.